yt/yt/server/lib/job_proxy/config.h

```diff
@@ -249,6 +249,10 @@ class TJobProxyInternalConfig
 
     std::optional<double> ContainerCpuLimit;
 
+    i64 ContainerMemoryLimit = 0;
```

Comment:
Do we need initialization here?

Reply:
This means unlimted by default. Probably it's better wrap it with std::optional.

Reply:
Yes